webpack.conf.js

```diff
@@ -140,7 +140,7 @@ module.exports = {
           })        }
       })
     ],
-    splitChunks: {
+    splitChunks: (argv._ && argv._.includes('bundle')) ? false : {
```

Comment:
I don't understand this - when does the build see `--bundle`? do you have some custom logic that runs webpack twice?

Reply:
i was also of the same opinion, but when i ran `gulp bundle` i saw the code being executed under splitChunk.

Reply:
Is it possible that what you saw is the code in `webpack.conf.js` run (which generates the config)? webpack itself should not run and no chunk-core should be generated. If there's a situation where just generating the config causes issues I'd like to understand what it is and fix it rather than attempting to conditionally turn it off - bundle is not the only situation where webpack doesn't run.